### PR TITLE
Eng 1669 cleanup

### PR DESCRIPTION
**Pull request by sblack4**

### CODE REVIEW

**Comment by sblack4** [comment]
The commit hook is using the wrong provider version. This is partially due to OUR HARDCODING THE TERRAFORM VERSION IN THE WORKFLOW FILES. End of rant.

**Comment by sblack4** [comment]
Ignore the failing workflow. it's #6

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 Rhythmic Technologies, Inc.
+Copyright (c) 2021 Rhythmic Technologies, Inc.
```
Comment (cdaniluk) [inline LICENSE:3]: 🎈

**Review by cdaniluk — APPROVED** [review]
This PR is astutely named.

**Pull request merged**